src/Controls/src/Core/BoxView/BoxView.Android.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace Microsoft.Maui.Controls
+{
+	public partial class BoxView
+	{
+		private protected override void OnHandlerChangedCore()
+		{
+			base.OnHandlerChangedCore();
+
+			if (Handler != null)
+			{
+				foreach (var prop in AndroidBatchPropertyMapper<BoxView, IViewHandler>.SkipList)
```

Comment:
These should all get propagated here
https://github.com/dotnet/maui/blob/6d12613747cf5b0e21287b714836e09847e351ef/src/Core/src/Handlers/View/ViewHandler.cs#L188-L192

Did you happen to discover why this isn't the case for BoxView?

Reply:
@PureWeen I added a new commit and reverted this one